controllers/cdapmaster_controller.go

```diff
@@ -122,8 +122,6 @@ func ApplyDefaults(resource interface{}) {
 	if spec.Config == nil {
 		spec.Config = make(map[string]string)
 	}
-	// Set the local data directory
-	spec.Config[confLocalDataDirKey] = confLocalDataDirVal
```

Comment:
set local data dir if absent in the cdap-cr's config section

Reply:
Done